public/app/features/browse-dashboards/components/CreateNewButton.tsx

```diff
@@ -12,9 +13,18 @@ interface Props {
 export function CreateNewButton({ inFolder }: Props) {
   const newMenu = (
     <Menu>
-      <MenuItem url={addFolderUidToUrl('/dashboard/new', inFolder)} label="Add new dashboard" />
-      <MenuItem url={addFolderUidToUrl('/dashboards/folder/new', inFolder)} label="Add new folder" />
-      <MenuItem url={addFolderUidToUrl('/dashboard/import', inFolder)} label="Import dashboard" />
+      <MenuItem
+        url={addFolderUidToUrl('/dashboard/new', inFolder)}
+        label={t('search.dashboard-actions.new-dashboard', 'New Dashboard')}
+      />
+      <MenuItem
+        url={addFolderUidToUrl('/dashboards/folder/new', inFolder)}
+        label={t('search.dashboard-actions.new-folder', 'New Folder')}
+      />
+      <MenuItem
+        url={addFolderUidToUrl('/dashboard/import', inFolder)}
+        label={t('search.dashboard-actions.import', 'Import')}
+      />
```

Comment:
Translation phrases should not exist in multiple files as it can cause desync/conflict issues. 

When we get https://github.com/grafana/grafana/pull/67193 in, you can move the phrase to `tempI18nPhrases.ts`, and use that in both places.

Reply:
@tskarhed rebase/merge main into this and move the phrases into `tempI18nPhrases.ts`